scripts/sct_deepgmseg.py

```diff
@@ -0,0 +1,68 @@
+import sys
```

Comment:
File header with something that says what this file does, and points to deegmseg for doc.

Reply:
There are many ways of adding header doc in sphinx, which format are we using ?

Reply:
I'm in favour of using this format: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Also we would need to add the license header:

```
Copyright (c) 2018 Polytechnique Montreal <www.neuro.polymtl.ca>
About the license: see the file LICENSE.TXT
```

Reply:
Please add the `#!/usr/bin/env python` on L1